src/Stripe.net/Entities/Charges/ChargePaymentMethodDetails.cs

```diff
@@ -12,14 +12,17 @@ public class ChargePaymentMethodDetails : StripeEntity<ChargePaymentMethodDetail
         public ChargePaymentMethodDetailsAchDebit AchDebit { get; set; }
 
         [JsonProperty("acss_debit")]
-        public ChargePaymentMethodDetailsAchDebit AcssDebit { get; set; }
+        public ChargePaymentMethodDetailsAcssDebit AcssDebit { get; set; }
```

Comment:
Fixed this one which could break deserialization. It's technically a breaking change but it would unblock users using ACSS Debit

Reply:
ah dang! good catch.